package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "rstools-web",
```

Comment:
Could you rename it to redstonetools-site?

Reply:
sure